registry/tracecat_registry/integrations/limacharlie.py

```diff
@@ -0,0 +1,47 @@
+"""Limacharlie authentication.
+
+Docs: https://docs.limacharlie.io/apidocs/introduction
+"""
+
+import httpx
+
+from tracecat_registry import RegistrySecret, registry, secrets
+
+LIMACHARLIE_AUTH_URL = "https://jwt.limacharlie.io"
```

Comment:
can we make this consistent - either all base API URLs defined as a global var like this, or defined inside the udf